primitives/linux-port/column.cpp

```diff
@@ -1763,7 +1772,8 @@ void filterColumnData(NewColRequestHeader* in, ColResultHeader* out, uint16_t* r
 
   // Bit patterns in srcArray[i] representing EMPTY and NULL values
   T emptyValue = getEmptyValue<T>(dataType);
-  T nullValue = getNullValue<T>(dataType);
+  // TODO why don't we get null value here like in _parseColumnFilter? It can be unsigned
```

Comment:
Please, provide examples where nullValue can be unsugned and type is signed, if you meant that.

As far as I can remember, colDataType is properly dispatched to specific C++ signed/unsigned type.

Reply:
Sorry, my bad, it was a TODO for myself (in some places in code null value is calculated with sign in mind, in others without it). Removed the comment